wiki/osu!academy/en.md

```diff
@@ -1,6 +1,6 @@
 ---
 needs_cleanup: true
-outdated: true
+outdated: false
```

Comment:
these are deleted, not set to false

Reply:
I see! This is my first time. Thank you!

Reply:
Done!